packages/block-library/src/heading/block.json

```diff
@@ -1,6 +1,14 @@
 {
 	"name": "core/heading",
 	"category": "common",
+	"__experimentalSelector": {
```

Comment:
Could we simplify and make __experimentalSelector always be a string? E.g: this selector would be "h1, h2, h3, h4, h5, h6".

Reply:
What is it mapping? Can you give a simple example. Selector has very broad meaning and can get easily confused with data selectors.

Reply:
@gziolo This PR is about generating some CSS per block based on some theme.json descriptions, for which we need to know what CSS selector to use for each block.

@jorgefilipecosta We need the ability to target individual elements (only the `h2`, etc).

Reply:
Should it be part of the `supports` object maybe? Or should we treat it as part of the block editor features? I guess we can start as is but I’m curious what are your thoughts. /cc @youknowriad @mtias

Reply:
I think specifying the CSS selectors for a block is not exactly the same as specifying a supports as in that case we don't enable anything we just say that when we need to target that block using CSS the specified selector should be used. Another problem with supports is that we don't define them in block.json and the server is not aware of the "supports" information.

Reply:
The plan was to replace `supports` with something else but it doesn’t look it’s happening so we should move it to `block.json`. I wouldn’t worry about that part. Let’s focus on other aspects 😃

Reply:
One thing we consider is to introduce `features` (or whatever else that better firs) property that would inherit `supports` and everything else we want to store in `theme.json`.

Reply:
I think this discussion is great to have and should be part of that of exploring how blocks should expose their selectors.